### PR TITLE
fix(curriculum): replace D.O.B. acronym in accessibility quiz

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #47830

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143931a113bb80c45546287.md
Comment (Supravisor) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143931a113bb80c45546287.md:10]: > Any ideas on how to incorporate the `sr-only` class for this lesson?

The first item under "Should Fix":

> "Question 1" and "Question 2" sr-only text be added to the beginning of their respective questions so that screen reader users navigating by tab stop will hear the question number as well as the question text.

_Originally posted by @bbsmooth in https://github.com/freeCodeCamp/freeCodeCamp/issues/47830#issuecomment-2116694307_

Will need to create another PR to address `sr-only` text for "Question 1" and "Question 2" headings, as separate PRs are required for the issue.

**Comment by Supravisor** [comment]
Is it okay for this PR to proceed without further changes?

Then the *should fix* can be applied on **Question 1** and **Question 2** using the `sr-only` class, in a separate PR.
Otherwise another issue may need to merge into this one.

**Comment by bbsmooth** [comment]
> Is it okay for this PR to proceed without further changes?

I would vote no.

"Although the `Date of Birth` text is descriptive, it can be problematic for visually impaired users."

This doesn't make sense. Date of Birth is not problematic for visually impaired users. It's not just a matter of changing D.O.B to Date of Birth. This entire step needs to be reworked.

**Comment by Supravisor** [comment]
> Is it okay for this PR to proceed without further changes?
> 
> Then the _should fix_ can be applied on **Question 1** and **Question 2** using the `sr-only` class, in a separate PR. Otherwise another issue may need to merge into this one.



> > Is it okay for this PR to proceed without further changes?
> 
> I would vote no.
> 
> "Although the `Date of Birth` text is descriptive, it can be problematic for visually impaired users."
> 
> This doesn't make sense. Date of Birth is not problematic for visually impaired users. Also, you wouldn't add visually hidden text that is exactly the same as the visible text. That's just unnecessary duplication for screen reader users.
> 
> It's not just a matter of changing D.O.B to Date of Birth. This entire step needs to be reworked.


My question was "Is it okay for this PR to proceed without further changes?" - MUST FIX No 4.  This way another PR can be made to add `sr-only` class for Question 1 and Question 2 - Should fix No 1.

naomi-lgbt requested that separate PRs are created to address ten identified issues.

Although several steps may require multiple rework, a way forward without merging one or more PRs looks tenuous.

Note: this PR already has one approval.

If this PR receives a second review approval, a pathway forward is outlined as follows:
1. MUST FIX No.3: The Question 1 and Question 2 text look like headings and should probably be marked up as headings.
2. Should Fix No. 1: "Question 1" and "Question 2" sr-only text be added to the beginning of their respective questions so that screen reader users navigating by tab stop will hear the question number as well as the question text.

**Comment by bbsmooth** [comment]
> My question was "Is it okay for this PR to proceed without further changes?"

And my answer is that it doesn't make sense to merge this PR as it is without further changes because it is wrong. 

"Although the `Date of Birth` text is descriptive, it can be problematic for visually impaired users. One way to get around such an issue is to add text only a screen reader can read."

This is just wrong. "Date of Birth" is not problematic for blind people. You would not add text that only a screen reader can read. If you ask me, the changes in this PR make this step worse and I'd rather leave the step as it is now.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
> naomi-lgbt requested that separate PRs are created to address ten identified issues.

Yes, but if the changes don't make sense to separate, then this doesn't apply.

In this case, it seems like we need to resolve multiple things at once.

- Instead of using D.O.B. for the label, use Date of Birth
- Change the screen reader text steps to apply `Question 1` and `Question 2` to the labels

**Comment by ojeytonwilliams** [comment]
Hey @Supravisor any thought on Naomi's comments?

**Comment by Supravisor** [comment]
I'll start on the suggestions in the next few days.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143956ed76ed60e012faa51.md
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143956ed76ed60e012faa51.md:10]:
```suggestion
Within the first `span` element, add the text `"Question"`.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143956ed76ed60e012faa51.md
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143956ed76ed60e012faa51.md:12]:
```suggestion
In the second `span` element, add the text `"Question"`.
```

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614396f7ae83f20ea6f9f4b3.md
Comment (naomi-lgbt) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614396f7ae83f20ea6f9f4b3.md:26]:
```suggestion
Use the above to define the `.sr-only` CSS rule.
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**
